src/DnDList.elm

```diff
@@ -512,25 +516,41 @@ info (Model model) =
         model
 
 
-subscriptions : (Msg -> msg) -> Model -> Sub msg
-subscriptions stepMsg (Model model) =
+subscriptions : (Msg -> msg) -> ((Json.Encode.Value -> msg) -> Sub msg) -> ((Json.Encode.Value -> msg) -> Sub msg) -> Model -> Sub msg
+subscriptions stepMsg onPointerMove onPointerUp (Model model) =
     case model of
         Nothing ->
             Sub.none
 
         Just _ ->
             Sub.batch
-                [ Browser.Events.onMouseMove
-                    (Internal.Common.Utils.decodeCoordinates
-                        |> Json.Decode.map (stepMsg << Drag)
+                [ 
+                --     Browser.Events.onMouseMove
+                --     (Internal.Common.Utils.decodeCoordinates
+                --         |> Json.Decode.map (stepMsg << Drag)
+                --     )
+                -- , Browser.Events.onMouseUp
+                --     (Json.Decode.succeed (stepMsg DragEnd))
+                -- , 
+                onPointerMove
+                    (\value ->
+                        -- let
+                        --     _ =
+                        --         Debug.log "onPointerMove" value
+                        -- in
+                        Json.Decode.decodeValue
+                            Internal.Common.Utils.decodeCoordinates
+                            value
+                            |> Result.withDefault (Position 0.0 0.0)
```

Comment:
It would be better to return Sub.non here, rather than defauting to Position 0.0 0.0